src/js/renderers/MultiEnumArrayControl.js

```diff
@@ -0,0 +1,114 @@
+import {
+	and,
+	hasType,
+	rankWith,
+	schemaMatches,
+	schemaSubPathMatches,
+	uiTypeIs
+} from "@jsonforms/core";
+  
+import { withJsonFormsMultiEnumProps } from "@jsonforms/react";
+import xorBy from "lodash/xorBy";
+import React from "react";
+
+import { 
+	FormTokenField,
+	__experimentalVStack as VStack,
+	Tooltip,	
+    FlexItem,
+} from '@wordpress/components';
+
+import styled from '@emotion/styled';
+
+export const GutenbergEnumArrayRenderer = ( {
+	schema,
+	visible,
+	errors,
+	path,
+	options,
+	data,
+	addItem,
+	removeItem,
+	handleChange: _handleChange,
+	description,
+	id,
+	label
+} ) => {
+
+	const HiddenLabelFormTokenFieldWrapper = styled( FlexItem )`
+		.components-form-token-field__label {
+			display: none;
+		}
+	`;
+
+	return !visible ? null : (
+		<>
+			<VStack justify="space-between">
+				<FlexItem>
+				{ description ? (
+					<Tooltip text={ description }>
+					<label htmlFor={ id }>
+						{ label }
+					</label>
+				</Tooltip>
+				) : ( 
+					<label htmlFor={ id }>
+						{ label }
+					</label> 
+				) }
+				</FlexItem>
+				<HiddenLabelFormTokenFieldWrapper>
+					<FormTokenField
+						onChange={ ( tokens ) => {
+							// Get distinct value between tokens and data (both ways) and check wether the value is belong to data or not, of yes, then use addItem, otherwise, use removeItem
+							const distinctTokens = xorBy( tokens, data );
+							if ( distinctTokens.length > 0 ) {
+								const value = distinctTokens[ 0 ];
+								if ( data.includes( value ) ) {
+									removeItem( path, value );
+								} else {
+									addItem( path, value );
+								}
+							}
+						} }
+						suggestions={ options.map( ( option ) => option.value ) }
+						value={ data }
+						__experimentalShowHowTo={ false }
+						__nextHasNoMarginBottom={ true }
+						__experimentalExpandOnFocus={ true }
+					/>
+				</HiddenLabelFormTokenFieldWrapper>
+			</VStack>
+		</>
+	)
+}
+
+const hasOneOfItems = schema =>
```

Comment:
Check the logic of these testers on https://github.com/bangank36/WP-Builder/issues/85#issuecomment-1661822757